web/pdf_page_view.js

```diff
@@ -563,23 +566,9 @@ class PDFPageView {
     }
     div.removeAttribute("data-loaded");
 
-    if (annotationLayerNode) {
-      // Hide the annotation layer until all elements are resized
-      // so they are not displayed on the already resized page.
-      this.annotationLayer.hide();
-    }
-    if (annotationEditorLayerNode) {
-      this.annotationEditorLayer.hide();
-    }
```

Comment:
What about pages that are still active in the `PDFPageViewBuffer` buffer, but not currently visible?

These annotation-related layers won't have e.g. their rotation updated *until* they become visible again, and it's not clear to me if that could cause them to overflow their respective page-containers and thus potentially "mess" with layout?